src/openzaak/utils/tasks.py

```diff
@@ -0,0 +1,791 @@
+        return self._processed
+
+    @processed.setter
+    def processed(self, value: bool):
+        self._processed = value
+
+    @property
+    def succeeded(self) -> bool:
+        return self.processed and self._succeeded
+
+    @succeeded.setter
+    def succeeded(self, value: bool):
+        self._succeeded = value
+
+    @property
+    def failed(self) -> bool:
+        return self.processed and not self.succeeded
+
+    @property
+    def has_instance(self) -> bool:
+        return bool(self.instance and self.instance.pk)
+
+    @property
+    def result(self):
+        if self.succeeded:
+            return ImportRowResultChoices.imported.label
+
+        return ImportRowResultChoices.not_imported.label
+
+    def as_serializer_data(self):
+        return {
+            "identificatie": self._identificatie,
+            "bronorganisatie": self.bronorganisatie,
+            "creatiedatum": self.creatiedatum,
+            "titel": self.titel,
+            "vertrouwelijkheidaanduiding": self._vertrouwelijkheidaanduiding,
+            "auteur": self.auteur,
+            "status": self._status,
+            "formaat": self._formaat,
+            "taal": self.taal,
+            "bestandsnaam": self._bestandsnaam,
+            "bestandsomvang": self.bestandsomvang,
+            "link": self._link,
+            "beschrijving": self._beschrijving,
+            "indicatie_gebruiksrecht": self.indicatie_gebruiksrecht,
+            "verschijningsvorm": self._verschijningsvorm,
+            "ondertekening": self.ondertekening,
+            "integriteit": self.integriteit,
+            "informatieobjecttype": self._informatieobjecttype,
+            "trefwoorden": self.trefwoorden,
+        }
+
+    def as_original(self):
+        return {
+            "uuid": self._uuid,
+            "identificatie": self._identificatie,
+            "bronorganisatie": self._bronorganisatie,
+            "creatiedatum": self._creatiedatum,
+            "titel": self._titel,
+            "vertrouwelijkheidaanduiding": self._vertrouwelijkheidaanduiding,
+            "auteur": self._auteur,
+            "status": self._status,
+            "formaat": self._formaat,
+            "taal": self._taal,
+            "bestandsnaam": self._bestandsnaam,
+            "bestandsomvang": self._bestandsomvang,
+            "bestandspad": self._bestandspad,
+            "link": self._link,
+            "beschrijving": self._beschrijving,
+            "indicatieGebruiksrecht": self._indicatie_gebruiksrecht,
+            "verschijningsvorm": self._verschijningsvorm,
+            "ondertekening.soort": self._ondertekening_soort,
+            "ondertekening.datum": self._ondertekening_datum,
+            "integriteit.algoritme": self._integriteit_algoritme,
+            "integriteit.waarde": self._integriteit_waarde,
+            "integriteit.datum": self._integriteit_datum,
+            "informatieobjecttype": self._informatieobjecttype,
+            "zaakId": self._zaak_id,
```

Comment:
I see this is part of the ticket, but what does `zaakId` mean here? Is it `zaak.identificatie` or `zaak.uuid`?

Reply:
The latter, it means the `UUID` of the zaak.